grout/models.py

```diff
@@ -24,14 +27,34 @@ class Meta(object):
         abstract = True
 
 
-class SchemaModel(GroutModel):
+class RecordType(GroutModel):
```

Comment:
I moved around some of the classes in this file to better approximate the hierarchical nature of the Grout data model (`RecordType -> RecordSchema -> Record`). This class, for instance, is now closer to the top.